plugins/coverimage.koplugin/main.lua

```diff
@@ -0,0 +1,279 @@
+                                                    text = is_path_ok_message,
+                                                    show_icon = true,
+                                                })
+                                            end
+                                        end
+                                        UIManager:close(sample_input)
+                                        menu:updateItems()
+                                    end,
+                                },
+                            }
+                        },
+                    }
+                    UIManager:show(sample_input)
+                    sample_input:onShowKeyboard()
+                end,
+            },
+            -- menu entry: enable
+            {
+                text = _("Save book cover"),
+                checked_func = function()
+                    return self:_enabled() and pathOk(self.cover_image_path)
+                end,
+                enabled_func = function()
+                    return self.cover_image_path ~= "" and pathOk(self.cover_image_path)
+                end,
+                callback = function()
+                    if self.cover_image_path ~= "" then
+                        self.enabled = not self.enabled
+                        G_reader_settings:saveSetting("cover_image_enabled", self.enabled)
+                        if self.enabled then
+                            self:createCoverImage(self.ui.doc_settings)
+                        else
+                            self:cleanUpImage()
+                        end
+                    end
+                end,
+            },
+            -- menu entry: exclude this cover
+            {
+                text = _("Exclude this book cover"),
+                checked_func = function()
+                    return self.ui and self.ui.doc_settings and self.ui.doc_settings:readSetting("exclude_cover_image") == true
+                end,
+                callback = function()
+                    if self.ui.doc_settings:readSetting("exclude_cover_image") == true then
+                        self.ui.doc_settings:saveSetting("exclude_cover_image", false)
+                        self:createCoverImage(self.ui.doc_settings)
+                    else
+                        self.ui.doc_settings:saveSetting("exclude_cover_image", true)
+                        self:cleanUpImage()
+                    end
+                    self.ui:saveSettings()
+                end,
+                separator = true,
+            },
+            -- menu entry: set fallback image
+            {
+                text = _("Set fallback image"),
+                checked_func = function()
+                    return lfs.attributes(self.cover_image_fallback_path, "mode") == "file"
+                end,
+                help_text =  _("File to use when no cover is wanted (found ???) or book is excluded.\nLeave this blank to turn off the fallback image."),
```

Comment:
@zwim Leftover draft of some sort, will you submit a PR to fix it?

Reply:
Oh, thank you.

Yes, we can delete the help_text, as in the dialog shows instructions.
But I have noticed, the wording of just this dialog is wrong:
 https://github.com/koreader/koreader/blob/master/plugins/coverimage.koplugin/main.lua#L325 
It should be: "Leave this empty to keep the cover when the document is closed.", as this is what the plugin actually does.

I do a PR a bit later, today.